interp/functions.go

```diff
@@ -405,25 +405,26 @@ func (p *interp) sprintf(format string, args []value) (string, error) {
 		case 's':
 			v = p.toString(a)
 		case 'd':
-			v = int64(a.num())
+			v = int(a.num())
```

Comment:
Can you please revert this change (and the similar `uint` one on line 412)? They should go back to `int64` and `uint64`. They are probably from an old branch.

Reply:
Commit [403d55d](https://github.com/benhoyt/goawk/pull/236/commits/403d55d22bfebfd0536c373ba682f39e62acd22c)